entity-framework/efcore-and-ef6/index.md

```diff
@@ -1,77 +1,74 @@
 | Constructors with parameters                          |          | 2.1                                   |
 | Property value conversions                            |          | 2.1                                   |
 | Mapped types with no keys                             |          | 2.1                                   |
 | Conventions                                           | Yes      | 1.0                                   |
-| Custom conventions                                    | Yes      | 1.0 (partial)                         |
+| Custom conventions                                    | Yes      | 1.0 (partial; [#214](https://github.com/dotnet/efcore/issues/214)) |
 | Data annotations                                      | Yes      | 1.0                                   |
 | Fluent API                                            | Yes      | 1.0                                   |
 | Inheritance: Table per hierarchy (TPH)                | Yes      | 1.0                                   |
-| Inheritance: Table per type (TPT)                     | Yes      |                                       |
-| Inheritance: Table per concrete class (TPC)           | Yes      |                                       |
+| Inheritance: Table per type (TPT)                     | Yes      | Planned for 5.0 ([#2266](https://github.com/dotnet/efcore/issues/2266)) |
+| Inheritance: Table per concrete class (TPC)           | Yes      | Stretch for 5.0 ([#3170](https://github.com/dotnet/efcore/issues/3170)) <sup>(1)</sup> |
 | Shadow state properties                               |          | 1.0                                   |
 | Alternate keys                                        |          | 1.0                                   |
-| Many-to-many without join entity                      | Yes      |                                       |
+| Many-to-many navigations                              | Yes      | Planned for 5.0 ([#19003](https://github.com/dotnet/efcore/issues/19003)) |
+| Many-to-many without join entity                      | Yes      | Backlogged ([#1368](https://github.com/dotnet/efcore/issues/1368)) |
 | Key generation: Database                              | Yes      | 1.0                                   |
 | Key generation: Client                                |          | 1.0                                   |
 | Complex/owned types                                   | Yes      | 2.0                                   |
 | Spatial data                                          | Yes      | 2.2                                   |
-| Graphical visualization of model                      | Yes      |                                       |
-| Graphical model editor                                | Yes      |                                       |
+| Graphical visualization of model                      | Yes      | No support planned <sup>(2)</sup>     |
```

Comment:
I'd move these (and the possibly the model format/create model items) to the bottom, after "core" features such as field mapping, table splitting...